modal/app.py

```diff
@@ -71,15 +56,15 @@ async def _create_all_objects(
     ):  # api_pb2.AppState.V
         """Create objects that have been defined but not created on the server."""
         resolver = Resolver(
-            self._client,
```

Comment:
I think you'll want to make sure you rebase and update this line: https://github.com/modal-labs/modal-client/blob/main/modal/app.py#L73

Reply:
Thanks!